addons/web/controllers/main.py

```diff
@@ -1555,8 +1549,7 @@ def get_app_url(self, req, app):
             action['views'] = [(False, u'form')]
 
         sakey = Session().save_session_action(action)
-        debug = '?debug' if req.debug else ''
-        return werkzeug.utils.redirect('/web{0}#sa={1}'.format(debug, sakey))
+        return werkzeug.utils.redirect('/web#sa={1}'.format(sakey))
```

Comment:
did you test ?

Reply:
Hm, obviously not since `{1}` does not exist, my bad, I will change that.
Anyway, I just had a quick look at this feature and it seems to not work anymore? Last working version I found is 11.0, later version will keep redirecting on /web and not module form view.

Reply:
Behavior fixed in saas-11.3 with https://github.com/odoo/odoo/commit/e9350993ca0b81df17308766d7e26d458bc39520